docs/src/pages/api/00_usage.md

```diff
@@ -2,86 +2,197 @@
 title: "Usage"
 ---
 
-### Node
+Import the Octokit constructor.
 
-Install with `npm install @octokit/rest`.
+For direct usage in browsers, download `octokit-rest.min.js` from the [latest release](https://github.com/octokit/rest.js/releases/latest) and import it using a `<script src="octokit-rest.min.js"></script>` tag.
```

Comment:
Why the change in focus from using npm to using the download in browsers?

Reply:
there is no change of focus, it’s a universal library that should run in all environments. I just updated the documentation with the content from the README.md